packages/backend/src/app.module.ts

```diff
@@ -12,11 +12,11 @@ import { DbModule } from './db/db.module';
       envFilePath:
         process.env.NODE_ENV === 'development'
           ? '.env.development.template'
-          : '.env.production',
+          : '.env.production.template',
```

Comment:
These should both just be using non-templates. I'm not sure why development was using the .template before. 

These envs will contain secrets at some point so we can't be using the templates as they are in the repo.

Reply:
Its the developers responsibility to ensure they create their own .envs based off the template.

Reply:
just change the filename?

Reply:
Nah, the templates stay, they just tell the developer what to put in their own `.env.development` and `.env.production`.

Reply:
not sure how that happened 😅

Reply:
this means that everyone who wants to build will have to manually copy `.env.xxx.template` to `.env.xxx`, seems a bit of a pain?

Reply:
I could make the template a fallback with a warning or something

Reply:
they only have to do it once. The template should not be used as a fallback. I might have a look later at changing it to be more like the FE but very low priority